core/state-machine/src/ext.rs

```diff
@@ -81,23 +80,33 @@ where
 	/// `storage_changes_root` is called matters + we need to remember additional
 	/// data at this moment (block number).
 	changes_trie_transaction: Option<(u64, MemoryDB<H>, H::Out)>,
+	/// Additional externalities for offchain workers.
+	/// If None the some methods from the trait might not supported.
```

Comment:
```suggestion
	/// If None some methods from the trait might not supported.
```
My english is bad but this is a typo isn't it ?